views/setDBpassword.ejs

```diff
@@ -0,0 +1,53 @@
+<style>
+    *{
+        text-align:center;
+    }
+    input {
+        height:40px;
+        margin-top:300px;   
+    }
+    button{
+        height: 40px;   
+    }
+    .container {
+        width: 450px;
+        margin: auto;
+    }
+    .body{
+        margin:10px;
+        height:700px;
+        border: 1px solid rgba(0,0,0,0.5);
+        background:rgba(220,220,220,0.5)
+    }
+</style>
+<main>
+    <div class="container">
+        <div class="header">
+            <h1 class="title">Database Setup</h1>
+        </div>
+        <div class="body">
+            <input class="dbPassword" placeholder="password">
+            <button class="setButton">Set</button>
+        </div>
+    </div>
+</main>
+<script>
+    const dbPassword = document.querySelector('.dbPassword')
+    const setButton = document.querySelector('.setButton')
+
+    setButton.addEventListener('click',()=>{
+        fetch('/api/users/:id',{
+            method:"PATCH",
+            headers:{
+                'Content-Type': 'application/json'
+            },
+            body:JSON.stringify({
+                password:dbPassword.value
+            })
+        }).then(r=>r.json()).then(r=>{
+            return location.href = '/databases'
+        })
+    })
+</script>
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/open-fonts@1.1.1/fonts/inter.min.css">
```

Comment:
Move this to the top of the file so the page doesn't flash with unstyled content
https://www.techrepublic.com/blog/web-designer/how-to-prevent-flash-of-unstyled-content-on-your-websites/